app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -323,7 +347,50 @@ private void updateMapToTrackPosition() {
 
                 mapboxMap.animateCamera(CameraUpdateFactory
                         .newCameraPosition(position), 1000);
+            }
+        }
+    }
+
+    /**
+     * Updates camera position according to list sheet status. If list sheet is collapsed, camera
+     * focus should be in the center. If list sheet is expanded, camera focus should be visible
+     * on the gap between list sheet and tab layout.
+     * @param isBottomListSheetExpanded
+     */
+    private void updateMapCameraAccordingToBottomSheet(boolean isBottomListSheetExpanded) {
```

Comment:
Codacy reports that this method is unused, could you double check? "Avoid unused private methods such as 'updateMapCameraAccordingToBottomSheet(boolean)'."

Reply:
Yes actually this method got obsolete after fix for #2018 .Although it was a doing a good job adjusting the job based on some calculations for adjusting map zoom level based on bottom sheet's state, but yes can be removed now.